### DISCUSSION TITLE
Is Vercel down?

### DISCUSSION THREAD

**Discussion by darthwade** [vercel/vercel#6244] [Help]
Hey,
I can't  access vercel.com and my sites on vercel as well, the platform is down or it's problem on my site?
Thanks!

**Comment by darthwade** [comment]
Resolved

**Comment by fu-sen** [comment]
I saw this and confirmed ▲Vercel, but it was normal.
Vercel Status: https://www.vercel-status.com/

But I saw another web service down and was working on it. I'm curious about your post.

**Comment by darthwade** [comment]
I found that all Vercel websites doesn't work from Orange Poland provider (https://www.orange.pl). From other internet providers everything is good
